lib/model-access-wrapper.js

```diff
@@ -607,7 +607,22 @@ class ModelAccessWrapper extends CrispHooks {
 	_checkDiffPermissions(permissions, doc, data) {
 		const grant = permissions.getTargetGrant(this.options.permissionsTarget, data);
 		const patch = Update.createFromDiff(doc.data, data);
-		const fields = _.pull(createUpdate(patch).getUpdatedFields(), '_id');
+		const schema = this.model.getSchema();
+		let fields = createUpdate(patch).getUpdatedFields();
+		let defaultedFields = [];
+
+		// Find defaulted fields
+		schema.traverseSchema({
```

Comment:
It may be faster/better to use `Schema#traverse()` here instead of `Schema#traverseSchema()`.  The former actually traverses an object alongside the schema, which is what you're doing here.